packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -561,20 +556,31 @@ export class SwapQuoter {
         } else {
             gasPrice = await this.getGasPriceEstimationOrThrowAsync();
         }
+
+        // If RFQT is enabled and `nativeExclusivelyRFQT` is set, then `ERC20BridgeSource.Native` should
+        // never be excluded.
+        if (
```

Comment:
⚠️ This is going to have some conflicts with https://github.com/0xProject/0x-monorepo/pull/2627 ⚠️